src/shared-components/toggle/style.js

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+
+import { COLORS, SPACING } from '../../constants';
+
+export const Container = styled.div`
+  position: relative;
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: ${SPACING.small} 0;
+
+  & > input {
+    display: none;
+  }
+`;
+
+export const Label = styled.span`
+  color: ${COLORS.primaryTint1};
+  margin: 0;
+  font-size: ${SPACING.small};
+  line-height: ${SPACING.base};
+  text-align: left;
+  cursor: pointer;
+  user-select: none;
+  margin-right: ${SPACING.xsmall};
+`;
+
+export const trackStyle = {
+  borderRadius: 100,
+  height: 24,
+  width: 40,
+};
+
+export const thumbStyle = {
```

Comment:
Also looks like the thumb needs to be nudged over 1px when it's toggled on and toggled off.

Reply:
I moved it closer to edge 1 pixel when active, moving 1 more pixel closer to each edge started to feel odd for me, take a look this is one more pixel to each side

![image](https://user-images.githubusercontent.com/11194969/56752058-d84c2880-675d-11e9-9d62-ef6dff1bb3e2.png)

Reply:
current
![image](https://user-images.githubusercontent.com/11194969/56752109-f154d980-675d-11e9-848e-3f9b9b08ad76.png)

Reply:
Okay let's go with whatever is marked Current. I don't know what I was seeing